scripts/create-secrets/k8sExecution.js

```diff
@@ -0,0 +1,320 @@
+const childProcess = require("child_process");
+const process = require("process");
+const chalk = require("chalk");
+const trim = require("lodash/trim");
+const {
+    getEnvVarInfo,
+    askIfCreateNamespace,
+    settingNameToEnvVarName
+} = require("./askQuestions");
+const Base64 = require("js-base64").Base64;
+const pwgen = require("./pwgen");
+
+const dbPasswordNames = [
+    "authorization-db",
+    "authorization-db-client",
+    "combined-db",
+    "combined-db-client",
+    "discussions-db",
+    "discussions-db-client",
+    "registry-db",
+    "registry-db-client",
+    "session-db",
+    "session-db-client"
+];
+
+let env = process.env;
+function k8sExecution(config, shouldNotAsk = false) {
+    try {
+        return doK8sExecution(config, shouldNotAsk);
+    } catch (e) {
+        return Promise.reject(e);
+    }
+}
+
+function doK8sExecution(config, shouldNotAsk = false) {
+    env = getEnvByClusterType(config);
+    let configData = Object.assign({}, config.all);
+    const ifAllowEnvVarOverride = configData["allow-env-override-settings"];
```

Comment:
Could this be `allowEnvVarOverride` ? So we don't end up with `if (if`

Reply:
Good point~ Will update this~😄